.typos.toml

```diff
@@ -1,64 +1,65 @@
 [files]
 extend-exclude = [
-    "*.js*", "*.json", "*.svg",
+    # files thta make no sense to be searched for typos
+    "*.jpg", "*.js*", "*.json", "*.mov", "*.png", "*.svg",
     "/static/jira",
     # external typo that must be replicated for technical correctness
     "content/blog/2020/09/2020-09-04-this-week-in-matrix-2020-09-04.md",
     # contains non-English
     "content/blog/2021/07/2021-07-02-this-week-in-matrix-2021-07-02.md",
     "content/blog/2021/09/2021-09-17-this-week-in-matrix-2021-09-17.md",
-    # Username wrongly detected as typo
-    "content/blog/2023/04/2023-04-14-this-week-in-matrix-2023-04-14.md",
-    "content/blog/2023/07/2023-07-28-twim.md",
-    # Tries to parse svg and image files
-    "static/blog/img/",
-    "static/js/",
-    # contains German
     "content/blog/2023/06/16-twim.md",
-    # German and hyphenated-word
-    "content/blog/2023/08/2023-08-11-twim.md"
+    "content/blog/2023/07/2023-07-28-twim.md",
+    # contains username that is also a common typo
+    "content/blog/2023/04/2023-04-14-this-week-in-matrix-2023-04-14.md",
 ]
 
 [default]
 extend-ignore-re = [
-    "\\{\\{ youtube_player\\(video_id=\"[^\"]+\"\\) \\}\\}"
+    # youtube embed in articles
+    "\\{\\{ youtube_player\\(video_id=\"[^\"]+\"\\) \\}\\}",
+
+    # slang
+    "tha'",
+    "Wee!",
+
+    # multi-word terms where an individual word is detected as a typo
+    # such as user names, project names, domain names
+    "Big Sur",
+    "Fuj'ehr",
+    "VaiTon",
+    "WeeChat",
+
+    "add-ons",
+    "json-clos",
+    "new-morg\\.png",
+    "whos-in-this-room",
+
+    "inex\\.rocks",
+    "raim\\.ist",
+    "loca\\.lt",
+    "programm\\.fr",
+    "supercable\\.onl",
+
+    # weird things hard or impossible to catch with heuristics
+    "ERRORs",
+    "H5atH-tbeRo", # can't get more general regex for otwsu.toml working
+    "KSVTtGsolBXtfsLTeAciXnBc\\.png",
+    "MESSAGEs",
+    "\\`Ther\\`",
+    "\\[ba685b7\\]",
+    "\\[96e9afe6\\]",
 ]
 
 [default.extend-words]
 requestor = "requestor" # valid spelling variant
```

Comment:
Hm why are some things moved to the regex while others are not?

Reply:
Nothing was moved to regex one-to-one, the things I moved are things that are not a single "word", ensuring that we still detect typos such as "ther" if it's not inside backticks and so on.

Reply:
Ohhhh. ok yeah makes sense